full-service/src/json_rpc/wallet.rs

```diff
@@ -806,12 +862,70 @@ where
                 txo_map,
             }
         }
+        JsonCommandRequest::get_txos_for_view_only_account {
+            account_id,
+            offset,
+            limit,
+        } => {
+            // TODO(cc) share logic between this and regular account get txos
```

Comment:
Let's not leave too many TODO comments.

Reply:
my plan was to address these before I merged this PR. They were TODOs for this PR and not TODOs to be left in the code